### DISCUSSION TITLE
Why & How do you use pH7Builder? 🤔

### DISCUSSION THREAD

**Discussion by pH-7** [pH7Software/pH7-Social-Dating-CMS#818] [General]
Feel free to describe your situation, your project and what you are doing.

I would love to learn more about you and what you do with pH7CMs solution! 🤗👍

**Comment by Hamzanazar12** [comment]
can you please guide me to add banners in ph7cms from settings/add banners?

**Comment by nightglazier** [comment]
hi im new to ph7cms im having trouble changing my headline code i cant find the right file. it comes up a red colour with an underline i need to get rid of the underline and change the colour. what file do i need to open in order for me to do this please.

**Comment by polynamaude** [comment]
@nightglazier Hi !
Just a question ? Do you think you are asking at the good place ? Can you read the title of this thread... ?

https://github.com/pH7Software/pH7-Social-Dating-CMS/wiki
https://ph7cms.com/doc/

There's a trick called basic rules in life and it's not something only cat, fish and butterfly have to obey....
Have you tried looking at the documentation ? Just one thing, what have you tried other than asking the question at a place where it says "What do you use pH7Builder for ?". At least if you'd talk a bit about what you are using the software for and after a series of messages then decide to ask a unrelated question, then I'd understand...

**Reply by nightglazier** [reply]
Hey sorry about that. I’ve had a look at the links you have sent I’m new to GitHub so I wasn’t sure where the right place to put my question in  . The links you sent I’ve already read those basically what it is I’m trying to do is edit the file for my promo text headline so I can change the colour and remove the underline I am using ph7cmsbuilder which I do like I’m just having some trouble locating a file I’m searched a lot of places just can’t seem to find the right file.

**Comment by polynamaude** [comment]
@nightglazier I give you a hint. Simply use the _**fgrep**_ command in a shell to search for the string you want to change. This will give you all the occurence of the string within the whole source code. This will allow you to find all you need.

You can also use the option in your IDE to do the same.

**Comment by Sonoma95SS** [comment]
If you're on Windows, I use Notepad++ to "Find in Files" to locate keywords in the files to find changes like this and then I make note in a spreadsheet so I'll know next time.
